docs/Lua API.rst

```diff
@@ -1004,6 +1004,21 @@ Fortress mode
   Resets the fortress mode sidebar menus and cursors to their default state. If
   ``pause`` is true, also pauses the game.
 
+* ``dfhack.gui.pauseRecenter(pos[,pause])``
+  ``dfhack.gui.pauseRecenter(x,y,z[,pause])``
+
+  Same as ``resetDwarfmodeView``, but also recenter if ``x`` isn't ``-30000``, and respects
+  RECENTER_INTERFACE_SHUTDOWN_MS (the delay before input is recognized when a recenter occurs) in DF's init.txt.
+
+* ``dfhack.gui.recenterViewscreen(pos[,zoom])``
+  ``dfhack.gui.recenterViewscreen(x,y,z[,zoom])``
+  ``dfhack.gui.recenterViewscreen([zoom])``
+
+  Recenter the view on a position using a specific zoom type. If no position is given,
+  recenter on ``df.global.cursor``. Zoom types are ``df.report_zoom_type`` (0 = Generic, 1 = Item, 2 = Unit),
```

Comment:
IMO we should not be listing the raw numeric values of enum items in the docs - the point of the names is that they are more stable than the raw values, so we should encourage people to pass in parameters by name by not providing numbers in the docs.

(Ideally the explanation of what the values mean would be in docs for this specific enum, but we don't have that yet, so I'm fine with it here instead.)